src/dtx/dtx_internal.h

```diff
@@ -234,4 +235,8 @@ enum dtx_status_handle_result {
 	DSHR_CORRUPT		= 5,
 };
 
+enum dtx_rpc_flags {
+	DRF_INITIAL_LEADER	= (1 << 0),
```

Comment:
a few comment maybe helpful, thx

Reply:
Sure, I will add comment in subsequent patch.